library_garage/db/load_data_to_db.rb

```diff
@@ -0,0 +1,70 @@
+require '../classes/author.rb'
+require '../classes/book.rb'
+require '../classes/reader.rb'
+require '../classes/order.rb'
+require '../classes/library.rb'
+require 'yaml'
+
+class Downloader
```

Comment:
We are using here 'load_data_to_db.rb' and class Downloader only for filling the database. It is used only once before finding the statistics. It is not used each time.